cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -207,12 +207,138 @@ def test_course_not_configured(self):
         self.assertEqual(self.client.get(self.url).status_code, 404)
 
 
+class VideoUploadPostTestsMixin(object):
+    """
+    Shared test cases for video post tests.
+    """
+    @override_settings(AWS_ACCESS_KEY_ID='test_key_id', AWS_SECRET_ACCESS_KEY='test_secret')
+    @patch('boto.s3.key.Key')
+    @patch('boto.s3.connection.S3Connection')
+    def test_post_success(self, mock_conn, mock_key):
+        files = [
+            {
+                'file_name': 'first.mp4',
+                'content_type': 'video/mp4',
+            },
+            {
+                'file_name': 'second.mp4',
+                'content_type': 'video/mp4',
+            },
+            {
+                'file_name': 'third.mov',
+                'content_type': 'video/quicktime',
+            },
+            {
+                'file_name': 'fourth.mp4',
+                'content_type': 'video/mp4',
+            },
+        ]
+
+        bucket = Mock()
```

Comment:
Can you add some comments here to educate other developers you are setting up mock for API return values? I feel like the logic here is a bit difficult to read for untrained eyes.